circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  node:
+    version: v0.10.40
+
+
+dependencies:
+  pre:
+    - npm install -g npm@3
+
+test:
+  override:
+    - ./node_modules/.bin/eslint --debug . --format tap | ./node_modules/.bin/tap-xunit > ${CIRCLE_TEST_REPORTS}/lint.xml && test ${PIPESTATUS[0]} -eq 0
```

Comment:
using `--format tap | tap-xunit` so lint results end up in `tests` tab (circleci)